### PR TITLE
Support customized attributes

### PR DESCRIPTION
**Pull request by lyweiwei**
In column config and row config, we allow people to pass in custom attributes (for the TH, TD and TR elements).

### CODE REVIEW

File: demos/vnext/rows/index.js [hunk 1]
```diff
@@ -50,15 +51,22 @@ window.gridView = pgrid.factory({ vnext: true }).create({
...
       type: 'data-rows',
       classes: {
-        redColor: true,
-        longer: (row) => { return row.ShipAddress.length > 15; },
-      }
+        'red-color': true,
+        'longer': row => (row.ShipAddress.length > 15),
+      },
+      attributes: {
+        'data-order-id': _.property('OrderID')
```
Comment (steins-z) [inline demos/vnext/rows/index.js]: better have a comma at the end of line

File: spec/integrated/vnext-spec/rows-spec.js [hunk 2]
```diff
@@ -35,45 +35,46 @@ let gridConfig = {
...
+        'data-title': ({ view }) => view.title,
       },
-    ]
+    }],
   },
 };
 
-let pgrid;
-let gridView;
+let pgrid, gridView;
```
Comment (steins-z) [inline spec/integrated/vnext-spec/rows-spec.js:69]: better to keep wrap when define variables?

**Review by steins-z — APPROVED** [review]

File: spec/integrated/vnext-spec/rows-spec.js (hunk 2 above)
Comment (lyweiwei) [reply spec/integrated/vnext-spec/rows-spec.js:69]: per the eslint rule we are using, if the variables are not initialized, we should put them in 1 line.

File: demos/vnext/rows/index.js (hunk 1 above)
Comment (lyweiwei) [reply demos/vnext/rows/index.js]: good catch

**Pull request merged**